app/frontend/src/submissions/components/SubmissionForm/Casings.vue

```diff
@@ -13,7 +13,17 @@ Licensed under the Apache License, Version 2.0 (the "License");
 */
 <template>
   <fieldset>
-    <legend :id="id">Casing Details</legend>
+    <b-row>
+      <b-col cols="12" lg="6">
+        <legend :id="id">Casing Details</legend>
+      </b-col>
+      <b-col cols="12" lg="6">
```

Comment:
I'd still like this to be a common component - instead of cut 'n paste - but it's really not critical right now.